<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shape Detection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>Picture</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>20</width>
      <height>521</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>271</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="input_tab">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QRadioButton" name="image_rButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="camera_rButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera</string>
      </property>
     </widget>
     <widget class="QLabel" name="add_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Address :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="address_lineEdit">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>50</y>
        <width>251</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="address_button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="open_button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>175</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QLabel" name="cam_label">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>141</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera Number :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cam_comboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>130</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="blue_slider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>245</y>
        <width>230</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>490</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="blue_label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>233</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>WHITE_BALANCE_BLUE_U</string>
      </property>
     </widget>
     <widget class="QSlider" name="red_slider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>277</y>
        <width>230</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>450</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="red_label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>267</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>WHITE_BALANCE_RED_V</string>
      </property>
     </widget>
     <widget class="QLabel" name="expo_label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>297</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>EXPOSURE</string>
      </property>
     </widget>
     <widget class="QSlider" name="exposure_slider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>307</y>
        <width>230</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="camSet_checkBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>203</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edit Camera Setting</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="xml_checkBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>336</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>use an XML file for Camera Calibration</string>
      </property>
     </widget>
     <widget class="QPushButton" name="xml_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xmlAdd_lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>360</y>
        <width>210</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="filter_tab">
     <attribute name="title">
      <string>Filters</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Missions</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="stall_button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>450</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stall!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
